docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.tables.scss

```diff
@@ -26,7 +26,8 @@
   top: 0;
   @include hb-global-color('border-color', 'white');
 
-  a {
+  a,
+  a:not([class]) {
```

Comment:
I can't think of a reason why `a` and `a:not([class])` styles for table headings would be any different, so let's refactor lines 38 - 41 into one declaration for table heading links.  So that we don't have `a:not([class])` declared twice (line 30 and 38).

And I think we should flip the colors on the links. We there isn't a link on a table heading, it is white. Links should also be white, then change to `secondary-highlight` on hover. This will make the color on all table headings consistent. Links are indicated by the text underline. And the hover state darkens slightly. Let me know if that makes sense.

Reply:
I have refactored how these were laid out, that was a great idea and a good design change for the color consistency.